abnormal-security/info.json

```diff
@@ -0,0 +1,1259 @@
+          "value": false
+        }
+      ],
+      "output_schema": {
+        "name": "",
+        "email": "",
+        "title": "",
+        "manager": ""
+      }
+    },
+    {
+      "operation": "get_employee_identity_analysis",
+      "annotation": "get_employee_identity_analysis",
+      "category": "investigation",
+      "title": "Get Employee Identity Analysis",
+      "description": "Retrieve employee identity analysis (Genome) data based on the email address that you have specified.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Email Address",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "emailAddress",
+          "tooltip": "Specify the email address of the employee whose identity analysis you want to retrieve from the Abnormal Security server.",
+          "description": "Specify the email address of the employee whose identity analysis you want to retrieve from the Abnormal Security server."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {
+        "data": [
+          {
+            "key": "",
+            "name": "",
+            "description": "",
+            "values": [
+              {
+                "value": "",
+                "percentage": "",
+                "total_count": ""
+              }
+            ]
+          }
+        ]
+      }
+    },
+    {
+      "operation": "get_employee_login_info",
+      "annotation": "get_employee_login_info",
+      "category": "investigation",
+      "title": "Get Employee Login Information",
+      "description": "Get employee login information for last 30 days in csv format.",
+      "enabled": true,
+      "parameters": [
+        {
+          "title": "Email Address",
+          "required": true,
+          "visible": true,
+          "editable": true,
+          "type": "text",
+          "name": "emailAddress",
+          "tooltip": "Specify the email address of the employee whose login information you want to retrieve from the Abnormal Security server.",
+          "description": "Specify the email address of the employee whose login information you want to retrieve from the Abnormal Security server."
+        },
+        {
+          "title": "Mock Data",
+          "required": false,
+          "visible": true,
+          "editable": true,
+          "type": "checkbox",
+          "name": "mock-data",
+          "tooltip": "Returns test data if set to True.",
+          "description": "Returns test data if set to True.",
+          "value": false
+        }
+      ],
+      "output_schema": {}
```

Comment:
Output Schema is missing here, CSV IRI info should be given as output.

Reply:
done